src/vm/wren_common.h

```diff
@@ -37,8 +37,8 @@
 // Defaults to on on supported compilers.
 #ifndef WREN_COMPUTED_GOTO
   #ifdef _MSC_VER
-    // No computed gotos in Visual Studio.
-    #define WREN_COMPUTED_GOTO 0
+    // No real computed gotos in Visual Studio. Use the pseudo version.
+    #define WREN_COMPUTED_GOTO 2
```

Comment:
Instead of numeric values, let's do separate defines, I'm thinking:

```c
WREN_DISPATCH_SWITCH
WREN_DISPATCH_COMPUTED_GOTO
WREN_DISPATCH_JUMP_TABLES
```

`SWITCH` is the 0 case here, `COMPUTED_GOTO` is 1, and `JUMP_TABLES` is 2.